varlink/tests/org.example.encoding.varlink

```diff
@@ -0,0 +1,27 @@
+# Example Varlink service
+interface org.example.encoding
+
+type State (
+  start: ?bool,
+  progress: ?int,
+  end: ?bool
```

Comment:
Nit: inconsistent indentation with the rest of the varlink file.

Reply:
Updated. To match the existing file it is now using 2 space indents throughout.